packages/react-components/review/stable/react-components.api.md

```diff
@@ -122,14 +122,21 @@ export interface CameraButtonStyles extends ControlBarButtonStyles {
     menuStyles?: Partial<CameraButtonContextualMenuStyles>;
 }
 
+// @internal
+export const _Caption: (props: _CaptionProps) => JSX.Element;
+
+// @internal
```

Comment:
Why do we put it here? onRenderAvatar should not be for each caption, it should be for the whole composite

Reply:
Back to this comment, it's okay to put it here, but we should bring it back to _CaptionsBannerProps 
Inheritance seems not necessary between these 2 components